kolibri/plugins/video_mp4_render/assets/src/views/index.vue

```diff
@@ -26,6 +26,8 @@
   const LangLookup = require('./languagelookup');
   const customButtons = require('./videojs-replay-forward-btns');
   const throttle = require('lodash/throttle');
+  const Lockr = require('lockr');
```

Comment:
`lockr` relies on the localStorage API in the backend, which may not be supported by older versions of Android 4: http://caniuse.com/#feat=namevalue-storage